modules/core/test/test_io.cpp

```diff
@@ -1569,6 +1569,32 @@ TEST(Core_InputOutput, FileStorage_json_null_object)
     fs.release();
 }
 
+TEST(Core_InputOutput, FileStorage_json_key_backslash)
+{
+     // equivalant to json {"\"":1,"\\":59,"Ġ\"":366,"\\\\":6852}
+    std::string test =
+        "{ "
+            "\"\\\"\":1,"
```

Comment:
Not clear for me. `"\"\\\"\":1,"` has a key `"\"` so equivalent in json is `{ "\"\\\"": 1}`

Reply:
I don't get where is not the same test results? The `test` string translates to `{"\"":1,"\\":59,"Ġ\"":366,"\\\\":6852}` listed in the comment after compiling.

Reply:
I got your point here. Looks like we need to actually eliminate back slash from the string before creating the node with this string as key.